refinery/user_files_manager/utils.py

```diff
@@ -1,4 +1,5 @@
 from django.contrib.auth.models import User
+from django.http import Http404
```

Comment:
Also, replace the line above with:
```python
from guardian.compat import get_user_model
```
then:
```python
User = get_user_model()
```
as described here: https://django-guardian.readthedocs.io/en/latest/configuration.html